framework/test/jest/unit/specs/modules/chain/dpos/verifyBlockForger.spec.js

```diff
@@ -31,6 +31,7 @@ describe('dpos.verifyBlockForger()', () => {
 					getActiveDelegatesForRound: jest
 						.fn()
 						.mockReturnValue(delegatePublicKeys),
+					// @todo create was stubbed but never used, create an issue to remove it or add a test case
```

Comment:
@todo left here

Reply:
It's not part of this PR. I just noticed it and wanted to raise it. Here is the issue: https://github.com/LiskHQ/lisk-sdk/issues/4306

Reply:
Thanks for pointing out!